BDWallet

bdwallet opened the pull request:
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A BTC address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://gist.github.com/Noc2/75bc58e8ce9b5d419ff883b0cf2b8c19).
- [x] The total funding amount of the project is below CHF 30k at the time of submission.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/w3f/Open-Grants-Program?pullRequest=115) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/w3f/Open-Grants-Program?pullRequest=115) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: bdwallet<br/>:x: linmingchang<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/w3f/Open-Grants-Program?pullRequest=115) it.</sub>

mmagician commented:
@bdwallet Thanks for your application.
I have a couple of questions regarding your grant:
1. What is a `cloud wallet`? Please elaborate on this, I'm not familiar with the term and I could not find references nor detailed explanations
2. Am I right in thinking that your app aims to provide wallet functionality (i.e. storing a token balance), as well as staking (nominating) functionality and "ratings"? There are already plenty of Polkadot wallets available. Staking is also possible via a number of providers (also, see a soon-to-be-terminated [Lunie](https://medium.com/luniehq/sunsetting-lunie-io-66d566c14ba1) project. How do you differentiate yourself from these projects?
3. Can you elaborate on the ratings please? Who can submit ratings and which projects can be rated (and perhaps you could provide a sample list for our Polkadot ecosystem)? How do you decide on the trustworthiness of a review?
4. How does project rating fit into a wallet application? To me it sounds like these are two completely different use cases, but perhaps I am missing something?

bdwallet commented:
> @bdwallet Thanks for your application.
> I have a couple of questions regarding your grant:
> 
> 1. What is a `cloud wallet`? Please elaborate on this, I'm not familiar with the term and I could not find references nor detailed explanations
> 2. Am I right in thinking that your app aims to provide wallet functionality (i.e. storing a token balance), as well as staking (nominating) functionality and "ratings"? There are already plenty of Polkadot wallets available. Staking is also possible via a number of providers (also, see a soon-to-be-terminated [Lunie](https://medium.com/luniehq/sunsetting-lunie-io-66d566c14ba1) project. How do you differentiate yourself from these projects?
> 3. Can you elaborate on the ratings please? Who can submit ratings and which projects can be rated (and perhaps you could provide a sample list for our Polkadot ecosystem)? How do you decide on the trustworthiness of a review?
> 4. How does project rating fit into a wallet application? To me it sounds like these are two completely different use cases, but perhaps I am missing something?

bdwallet commented:
@mmagician Thank you for your comments. The following is the answer to your question

> What is a `cloud wallet`? Please elaborate on this, I'm not familiar with the term and I could not find references nor detailed explanations

Cloud wallets can be considered as wallet implementations with partial centralization features, keeping their cryptocurrency for users, similar to "PayPal". Cloud wallets exist to reduce users who have no experience in using cryptocurrency wallets. They don’t need to know any implementation details about the wallet, such as mnemonic words, secret keys, etc., users can directly transfer and receive encrypted currency through the account registered in our cloud wallet (the account may be a mobile phone number or email) And use other services, it is more friendly to novice users. According to our research, many users in China have this requirement. They need a simple and easy-to-use wallet and don't need to record mnemonics and secret keys by themselves.

At the same time, we also have HD wallet to choose from to meet the needs of high-end users.

> Am I right in thinking that your app aims to provide wallet functionality (i.e. storing a token balance), as well as staking (nominating) functionality and "ratings"? There are already plenty of Polkadot wallets available. Staking is also possible via a number of providers (also, see a soon-to-be-terminated [Lunie](https://medium.com/luniehq/sunsetting-lunie-io-66d566c14ba1) project. How do you differentiate yourself from these projects?

Most of the cryptocurrency users in China do not have basic blockchain knowledge, and do not know how to participate in node construction, mortgage, voting, and nomination operations. BDWallet provides lower operating thresholds for users, compared to other Polkadot Wallet, users only need to care about the interest rate that can be obtained when participating in staking in BDWallet. We help users implement investment details.

At the same time, BDWallet pioneered the shared staking model. When user A invites user B to participate in DOT staking, user A can obtain corresponding HZT token rewards. HZT token is the governance currency issued by our wallet. Through this incentive model, more users will spontaneously invite DOT holders to participate in DOT Staking, so that more people will participate in the ecological construction of the DOT community.

> Can you elaborate on the ratings please? Who can submit ratings and which projects can be rated (and perhaps you could provide a sample list for our Polkadot ecosystem)? How do you decide on the trustworthiness of a review?

We have a professional review team to look for high-quality projects on the market and put them on the rating database. Users or project issuers can also submit project information to us, which can be included in the project database after passing the review.
At the same time, we have a team of professional reviewers to score and rate the items in the rating library, as well as to post comments. The reviewer is a professional blockchain practitioner or cryptocurrency investor. It takes 3 years of industry experience and more than 500,000 encrypted assets to be eligible to serve as a reviewer, and provide relevant certificates to the BD team for review. Provide this rigorous screening mechanism to ensure that each reviewer is professional and to ensure fair and impartial scoring and rating of projects. Of course, the reason why reviewers will actively participate in our project review is because we will give a certain amount of HZT token rewards for high-quality reviews.

> How does project rating fit into a wallet application? To me it sounds like these are two completely different use cases, but perhaps I am missing something?

Combining ratings and wallets is where our BDWallet is innovative. We provide users with professional project ratings. Users can view the project profile, reviewer evaluations, and reviewer ratings of related currencies through our rating library, so as to deepen users’ understanding of the project and avoid users not knowing the project. The situation makes blind investment decisions.

At the same time, users can leave a comment under each rating, and other users can also see the message, so that users can increase their stickiness and frequency of use of the wallet. Users not only use the wallet as a tool, but also a channel for obtaining and understanding project information.

mmagician commented:
@bdwallet That sounds indeed quite interesting. I see the part of having a selected team of experts as a bit fragile and non-transparent, but perhaps the selection mechanism could some day be replaced by on-chain expert selection?

Could you please integrate the above comments into the application?

Also, since this is a Polkadot specific grant, unfortunately we won't be able to fund Milestone 2.4. Therefore, may I please ask you to reduce the price of that milestone?

Thanks!

bdwallet commented:
@mmagician thank you very much for your suggestion. 
In future plans, we consider voting on the chain to elect our reviewers. In this way, we believe that our rating mechanism will be more transparent and credible.
I have incorporated related discussions into the application, and you can check the changes through the commit log.
And I have removed Milestone 2.4 and adjusted the cost of Milestone 2 from 0.7 BTC to 0.55 BTC.

Noc2 commented:
Thanks for the application. Looks interesting. Could you also integrate the shared staking model into the deliverables? Btw. I think the “cloud wallet” would be especially useful in combination with some sort of faucet for tokens to onboard users very easily (Obviously it needs to be resistant against sybil attacks) .

bdwallet commented:
@Noc2 Thanks for your advice.
I have added shared staking to milestone 3.3. At this stage, our wallet functions have been fully completed. With the distribution capabilities of shared staking, we can better promote the use of wallets and allow users to participate in Polkadot staking. The first stage of sharing staking adopts the WeChat sharing model, because WeChat has a huge user base, especially in China, and more invitation channels will be provided in the future.

Noc2 reviewed (approved):
Thanks for the update. Looks good to me.

semuelle commented:
This grant has been terminated due to inactivity.
